sources/packages/web/src/composables/useAssessment.ts

```diff
@@ -0,0 +1,42 @@
+import { StatusChipTypes } from "@/components/generic/StatusChip.models";
+import {
+  ScholasticStandingStatus,
+  StudentAppealStatus,
+  AssessmentHistoryStatus,
+} from "@/types";
+
+export function useAssessment() {
+  const mapRequestAssessmentChipStatus = (
+    status: StudentAppealStatus | ScholasticStandingStatus,
+  ): StatusChipTypes => {
+    switch (status) {
```

Comment:
This could be simplified as below:
```typescript
 switch (status) {
  case StudentAppealStatus.Pending:
  case ScholasticStandingStatus.Pending:
    return StatusChipTypes.Warning;
  case StudentAppealStatus.Declined:
  case ScholasticStandingStatus.Declined:
    return StatusChipTypes.Error;
  default:
    return StatusChipTypes.Inactive;
}
```